src/etc/tidy.py

```diff
@@ -146,6 +148,9 @@ def interesting_file(f):
                     if not hsh.startswith(rev):
                         report_err("snapshot out of date (" + date
                             + "): " + line)
+                match = re.match(r'^.*#!\[unstable.*\]', line)
+                if match:
+                    needs_unstable_attr.discard(filename)
                 else:
```

Comment:
I'm not sure what you mean? Perhaps I expressed myself poorly: This `else` used to belong to the pre-existing `if match:` in line 145, but now the `else` belongs to the `if match:` in line 152. Thus, before the `else` was executed if `re.match(r'^.*//\s*SNAP\s+(\w+)', line)` returned `None`, and now it's executed if `re.match(r'^.*#!\[unstable.*\]', line)` returns `None`. I *think* this would cause tidy to complain about *every* SNAP line even if the above regex matched it.

It appears we don't have any SNAP lines at the moment, which explains why the Travis logs aren't full of such warnings.